src/drivers/Kconfig

```diff
@@ -96,6 +96,13 @@ endmenu # "Decimation factors"
 
 endif # CAVS_DMIC
 
+config CAVS_SSP
+	bool "Intel cAVS SSP driver"
+	depends on CAVS
+	default n
+	help
+	  Select this to enable Intel cAVS SSP driver.
```

Comment:
Best to add more detail about SSP. i.e. supported formats as this will make it obvious it's needed for I2S etc.

Reply:
Could you please clarify what exactly should I put there? I have added full SSP name but not sure about "supported formats".